spring-integration-file/src/main/java/org/springframework/integration/file/config/AbstractRemoteFileInboundChannelAdapterParser.java

```diff
@@ -49,7 +51,8 @@ protected final BeanMetadataElement parseSource(Element element, ParserContext p
 
 		// configure the InboundFileSynchronizer properties
 		BeanDefinition expressionDef = IntegrationNamespaceUtils.createExpressionDefinitionFromValueOrExpression(
-				"remote-directory", "remote-directory-expression", parserContext, element, false);
+				"remote-directory", "remote-directory-expression", parserContext,
+				element, false);
```

Comment:
I am just wondering what drove this change - unnecessary reformatting causes loss of blame.